addons/copy-message-link/profile.js

```diff
@@ -0,0 +1,44 @@
+export default async function ({
+  addon,
+  global,
+  console
+}) {
+  await addon.tab.waitForElement("div.comment", {
+    markAsSeen: true
+  });
+  if (addon.settings.get('preferred') === 'reply') {
+    document.querySelectorAll('div.comment').forEach((elem) => {
+      let newElem = document.createElement('a');
+      newElem.className = 'reply';
+      newElem.textContent = 'Copy Link';
+      newElem.onclick = (e) => {
+        let newElem2 = document.createElement('textarea');
+        newElem2.value = `${location.href.split('#')[0]}#${e.target.parentElement.parentElement.parentElement.id}`;
```

Comment:
Would be cool if it used proper casing for the username - URL might say `world_languaGES` but it should instead be `World_Languages`